x-pack/plugins/reporting/server/plugin.ts

```diff
@@ -117,6 +117,7 @@ export class ReportingPlugin
         savedObjects,
         uiSettings,
         store,
+        security: core.security,
```

Comment:
I'd prefer to use `securityService` for naming, to distinguish this from the deprecated APIs
```suggestion
        securityService: core.security,
```